tools/validate_configs/golden_copies/configs/text_escape.yaml

```diff
@@ -20,7 +20,7 @@ vars:
   deployment_name: text_escape
   zone: us-east4-c
   labels:
-    red: \((blue))
+    red: blue
```

Comment:
@mr0re1 I think the intent of this test was to ensure that we could escape strings that look like literals. So, "red: blue" isn't helping us much here and we should probably delete it. WDYT?

Reply:
I've changed the test to have international characters in the label name and label value, to ensure that works as expected.

There might still be some value in keeping the test for escaping chars, because ghpc validation does not currently cover labels passed directly to modules.  Still interested to see what @mr0re1 has to say though.